apps/ui/src/components/Modal/Votes.vue

```diff
@@ -48,6 +48,14 @@ async function handleEndReached() {
   loadingMore.value = false;
 }
 
+function getChoiceText(proposalType: VoteType, choice: Choice) {
```

Comment:
This isn't `Choice`, it's either `number | number[]`.

Also we are now duplicating it in two files, can we move it to helpers file like there?
https://github.com/snapshot-labs/sx-monorepo/pull/174#issuecomment-1997578149